src/main/java/vdm/DemoEditor.java

```diff
@@ -196,7 +196,8 @@ public void actionPerformed(ActionEvent e) {
 
 		@Override
 		public void actionPerformed(ActionEvent e) {
-			model.clear();
+            model = new TickTableModel(view);
+            view.getTableTicks().setModel(model);
```

Comment:
Thank you for noticing this, but doing this disables the comboBoxes due to editors being reset.

Combo stays up because the editing is not stopped. We should do this fix instead:
```java
view.getTableTicks().getCellEditor().stopCellEditing();
model.clear();
```
This must also be added around Line 115:

```java
		view.getBtnDeleteSelectedTick().addActionListener(e -> {
			view.getTableTicks().getCellEditor().stopCellEditing();
```